client/assets/scss/colors.scss

```diff
@@ -1,4 +1,32 @@
-$blue: rgba(52, 88, 170, 1);
-$grey: rgba(106, 106, 106, 1);
-$red: rgba(204, 0, 0, 1);
-$white: rgba(255, 255, 255, 1);
\ No newline at end of file
+//Instructions:
```

Comment:
There's no need for instructions. We talked offline about moving these to a `contribution.md` but I don't believe that's required also. All code should go through code review anyway so it should be caught if it's not to standard.